specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -739,8 +1200,7 @@
           "description": "Long summary phrase for the next 120 minutes. Phrase length is 60+ characters."
         },
         "iconCode": {
-          "type": "integer",
-          "format": "int32",
+          "$ref": "#/definitions/IconCode",
           "description": "Numeric value representing an image that displays the `iconPhrase`. Please refer to [Weather Service Concepts](https://aka.ms/AzureMapsWeatherConcepts) for details."
```

Comment:
This `description` can be deleted since it is also in the `IconCode` definition.